gh-pages-stub/public/js/search.js

```diff
@@ -0,0 +1,93 @@
+(function() {
+
+  function isBlank(str) {
```

Comment:
can we have something like utils and move this logic to the utils? 
i think this function can be reused in several different places